config/locales/en.yml

```diff
@@ -58,6 +58,18 @@ en:
     update:
       success: "Confirmation updated."
 
+  cyclists:
+    new:
+      header: "Invite a Cyclist"
+    show:
+      header: "Cyclist: %{name}"
+      instructions:
+        >
+        A cyclist account has been created for %{email}.
+
+        Instruct them to visit the following link to reset their password
+        and gain access to the latest pickup checklist.
```

Comment:
The header has their name, which I had hoped was sufficient.

I'm going to punt on most copy decisions, though.

Deferring to @corwinharrell or @desi